openc3-cosmos-init/plugins/packages/openc3-tool-common/src/components/Openc3Screen.vue

```diff
@@ -720,6 +724,37 @@ export default {
         })
       })
     },
+    update: function () {
+      if (this.screenItems.length !== 0 && this.errors.length === 0) {
+        this.api
+          .get_tlm_values(this.screenItems, this.staleTime)
+          .then((data) => {
+            this.updateValues(data)
+          })
+          .catch((error) => {
+            this.errors.push({
+              type: 'usage',
+              message: error.message,
+              time: new Date().getTime(),
+            })
+          })
+      }
+    },
+    updateValues: function (values) {
+      this.updateCounter += 1
+      for (let i = 0; i < values.length; i++) {
+        values[i].push(this.updateCounter)
```

Comment:
What is the point of this updateCounter? It looks like it gets set to 0 initially and then just increments forever?

Reply:
I believe it drives the aging logic